### PR TITLE
Updated version to 3.5 everywhere and created a script to use in the future

### PR DESCRIPTION
**Pull request by deepakaravindr**
1. Updated version from 3.4.0 to 3.5.0 on the vsix manifest version
   and NuGetPackage.cs
2. Simplified common.props file to have the
   SemanticVersion as a single property.
3. Created an UpdateVersion script that can be used to update NuGet version 
   when creating new releases.

@yishaigalatzer @emgarten @alpaix

### CODE REVIEW

File: scripts/utils/UpdateNuGetVersion.ps1 [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    }
+}
+
+Write-Host "Changing directory to $nuGetRoot.."
+cd $nuGetRoot
+
+ls -r project.json | ?{ ReplaceNuGetVersion($_.FullName) }
+
+$vsixManifestFile = Join-Path $nuGetRoot ".\src\NuGet.Clients\VsExtension\source.extension.vsixmanifest"
+ReplaceNuGetVersion($vsixManifestFile)
```
Comment (alpaix) [inline scripts/utils/UpdateNuGetVersion.ps1]: `.\` is redundant.

File: scripts/utils/UpdateNuGetVersion.ps1 [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        $d = $c | %{
+            $_.Replace($oldVersion, $newVersion)
+        }
+
+        Set-Content $file $d | Out-Null
+    }
+}
+
+Write-Host "Changing directory to $nuGetRoot.."
+cd $nuGetRoot
```
Comment (alpaix) [inline scripts/utils/UpdateNuGetVersion.ps1]: Is this needed? Script passes full paths everywhere.

File: build/common.props [hunk 3]
```diff
@@ -31,9 +31,7 @@
 		  Semantic Version. See http://semver.org for full details.
 		  Update for every public release.
 		-->
-		<SemanticVersionMajor>3</SemanticVersionMajor>
-		<SemanticVersionMinor>5</SemanticVersionMinor>
-		<SemanticVersionPatch>0</SemanticVersionPatch>
+		<SemanticVersion>3.5.0</SemanticVersion>
```
Comment (alpaix) [inline build/common.props:34]: :+1:

**Comment by emgarten** [comment]
:shipit:

File: scripts/utils/UpdateNuGetVersion.ps1 (hunk 1 above)
Comment (deepakaravindr) [inline scripts/utils/UpdateNuGetVersion.ps1]: Done

File: scripts/utils/UpdateNuGetVersion.ps1 (hunk 2 above)
Comment (deepakaravindr) [inline scripts/utils/UpdateNuGetVersion.ps1]: True. Removed it

**Comment by alpaix** [comment]
:shipit:

**Comment by deepakaravindr** [comment]
Merged with https://github.com/NuGet/NuGet.Client/commit/3e7620feae5f9ca7c3410daf10b6701ab71f40de

**Pull request closed without merging**